src/services/BidService.ts

```diff
@@ -0,0 +1,145 @@
+import JobContext from 'decentraland-gatsby/dist/entities/Job/context'
+
+import BidModel from '../entities/Bid/model'
+import { BidStatus, UnpublishedBid } from '../entities/Bid/types'
+import { GATSBY_GRANT_VP_THRESHOLD } from '../entities/Grant/constants'
+import ProposalModel from '../entities/Proposal/model'
+import { ProposalType } from '../entities/Proposal/types'
+import { DEFAULT_CHOICES } from '../entities/Proposal/utils'
+import Time from '../utils/date/Time'
+import { ErrorCategory } from '../utils/errorCategories'
+import { isProdEnv } from '../utils/governanceEnvs'
+import { getHighBudgetVpThreshold } from '../utils/projects'
+
+import { ErrorService } from './ErrorService'
+import { ProposalService } from './ProposalService'
+
+const MINIMUM_BIDS_TO_PUBLISH = Number(process.env.MINIMUM_BIDS_TO_PUBLISH || 0)
+export default class BidService {
+  static async createBid(
+    linked_proposal_id: string,
+    author_address: string,
+    bid_proposal_data: UnpublishedBid['bid_proposal_data']
+  ) {
+    const bidsSubmissionWindow = process.env.SUBMISSION_WINDOW_DURATION_BID
```

Comment:
You can write this like 
`const bidsSubmissionWindow = process.env.SUBMISSION_WINDOW_DURATION_BID && Number(process.env.SUBMISSION_WINDOW_DURATION_BID)`

Reply:
It seems it's not exactly the same 🤔

![image](https://github.com/decentraland/governance/assets/45410089/f08bc572-c99a-49c1-89dd-647741376d0b)
![image](https://github.com/decentraland/governance/assets/45410089/2746f0fd-7fb8-43f9-ab3d-6d67c4642e8c)